site/en/articles/css-nesting/index.md

```diff
@@ -0,0 +1,704 @@
+---
+layout: layouts/blog-post.njk
+title: CSS Nesting
+description: >
+  One of our favorite CSS preprocessor features is now built into the language: nesting style rules.
+subhead: >
+  One of our favorite CSS preprocessor features is now built into the language: nesting style rules.
+date: 2023-03-07
+authors:
+  - argyle
+tags:
+  - css
+  - chrome-112
+hero: image/vS06HQ1YTsbMKSFTIPl2iogUQP73/zyWAIWcvGAdAWDikew8n.jpg
+alt: >
+  Two colorful origami cranes.
+---
+
+Before nesting, every selector needed to be explicitly declared, separately from
+one another. This leads to repetition, stylesheet bulk and a scattered authoring
+experience.
+
+{% Compare 'worse', 'Before' %}
+```css
+.nesting {
+  color: hotpink;
+}
+
+.nesting > .is {
+  color: rebeccapurple;
+}
+
+.nesting > .is > .awesome {
+  color: deeppink;
+}
+```
+{% endCompare %}
+
+After [nesting](https://www.w3.org/TR/css-nesting-1/), selectors can be
+continued and related style rules to it can be grouped within.
+
+{% Compare 'better', 'After' %}
+```css
+.nesting {
+  color: hotpink;
+
+  > .is {
+    color: rebeccapurple;
+
+    > .awesome {
+      color: deeppink;
+    }
+  }
+}
+```
+{% endCompare %}
+
+[Try this in browser](https://codepen.io/web-dot-dev/pen/RwYLRdr).
+
+Nesting helps developers by reducing the need to repeat selectors while also
+co-locating style rules for related elements. It can also help styles match the
+HTML they target. If the `.nesting` component in the previous example was
+removed from the project, you could delete the entire group instead of searching
+files for related selector instances.
+
+**Nesting can help with:**
+- Organization
+- Reducing file size
+- Refactoring
+
+{% BrowserCompat 'css.nesting' %}
```

Comment:
I don't think this is in BCD right now, trying to find out where it ought to go, maybe attached to the `&` selector. We can add it in later though.

Reply:
confirmed it's not, this is a stub. 

i filed a request/issue for it back in jan https://github.com/mdn/mdn/issues/359

agree we can add it later. 

```suggestion
<!-- {% BrowserCompat 'css.nesting' %} -->
```